dist/web3-light.js

```diff
@@ -1429,22 +1429,26 @@ web3.version.api = version.version;
 web3.eth = {};
 
 /*jshint maxparams:4 */
-web3.eth.filter = function (fil, eventParams, options, formatter) {
+web3.eth.filter = function (fil, eventParams, options, callback) {
+
+    if (utils.isFunction(arguments[arguments.length - 1])) {
+        callback = arguments[arguments.length - 1];
```

Comment:
ok :)

If we remove `if (fil._isEvent...` (see comment below) we will be able to remove these 2 redundant params (`eventParams` and `options`) and this line :)